docs/contents/services.md

```diff
@@ -1,204 +1,223 @@
+| AWS [Lambda](#aws-lambda) | [`BUILD`](https://github.com/aws-controllers-k8s/community/issues/238) | | https://github.com/aws-controllers-k8s/community/milestone/10
+| AWS [Kinesis](#aws-kinesis) | [`PROPOSED`](https://github.com/aws-controllers-k8s/community/issues/235) | |
+| Amazon [KMS](#amazon-kms) | [`BUILD`](https://github.com/aws-controllers-k8s/community/issues/491) | | https://github.com/aws-controllers-k8s/community/milestone/18
+| Amazon [MQ](#amazon-mq) | [`BUILD`](https://github.com/aws-controllers-k8s/community/issues/390) | | https://github.com/aws-controllers-k8s/community/milestone/12
+| Amazon [MSK](#amazon-msk) | [`PLANNED`](https://github.com/aws-controllers-k8s/community/issues/348) | | https://github.com/aws-controllers-k8s/community/milestone/13
+| Amazon [RDS](#amazon-rds) | [`BUILD`](https://github.com/aws-controllers-k8s/community/issues/237) | | https://github.com/aws-controllers-k8s/community/milestone/8
+| Amazon [Route53](#amazon-route53) | [`PROPOSED`](https://github.com/aws-controllers-k8s/community/issues/480) | |
+| Amazon [SageMaker](#amazon-sagemaker) | [`BUILD`](https://github.com/aws-controllers-k8s/community/issues/385) | | https://github.com/aws-controllers-k8s/community/milestone/11
+| Amazon [SNS](#amazon-sns) | [`RELEASED`](https://github.com/aws-controllers-k8s/community/issues/202) | `PREVIEW` |
+| Amazon [SQS](#amazon-sqs) | [`BUILD`](https://github.com/aws-controllers-k8s/community/issues/205) | | https://github.com/aws-controllers-k8s/community/milestone/6
+| AWS [Step Functions](#aws-step-functions) | [`RELEASED`](https://github.com/aws-controllers-k8s/community/issues/239) | `PREVIEW` |
+| Amazon [S3](#amazon-s3) | [`RELEASED`](https://github.com/aws-controllers-k8s/community/issues/204) | `PREVIEW` |
 
 !!! note "Don't see a service listed?"
     If you don't see a particular AWS service listed, feel free to
-    [propose it](https://github.com/aws/aws-controllers-k8s/issues/new?labels=Service+Controller&template=propose_new_controller.md&title=%5Bname%5D+service+controller)!
+    [propose it](https://github.com/aws-controllers-k8s/community/issues/new?labels=Service+Controller&template=propose_new_controller.md&title=%5Bname%5D+service+controller)!
 
 ## Amazon ACM
 
-* Proposed: https://github.com/aws/aws-controllers-k8s/issues/482
-* Current release status: `PROPOSED`
+* Proposed: https://github.com/aws-controllers-k8s/community/issues/482
+* Current build stage: `PROPOSED`
 * AWS service documentation: https://aws.amazon.com/acm/
 
 ## Amazon API Gateway v2
 
-* Proposed: https://github.com/aws/aws-controllers-k8s/issues/207
-* Current release status: `DEVELOPER PREVIEW`
-* Next milestone: [`BETA`](https://github.com/aws/aws-controllers-k8s/milestone/15)
+* Proposed: https://github.com/aws-controllers-k8s/community/issues/207
+* Current build stage: `RELEASED`
+* Current maintenance phase: `PREVIEW`
+* Next milestone: https://github.com/aws-controllers-k8s/community/milestone/15
```

Comment:
Will all this information (Next Milestone, Curreny Phase, etc) be pulled and updated automatically?

Reply:
eventually yes, I'd like it to be.